synedrion/src/paillier/rsa.rs

```diff
@@ -262,13 +287,25 @@ impl<P: PaillierParams> PublicModulus<P> {
         let modulus = self.modulus_nonzero();
         loop {
             let r = P::Uint::random_mod(rng, &modulus);
-            if r.gcd(&self.modulus.0) == P::Uint::one() {
+            if r.is_invertible(&self.modulus.0) {
+                return r;
+            }
+        }
+    }
+
+    /// Returns a number in range $[0, N)$ such that it is invertible modulo $N$, in Montgomery form,
```

Comment:
Here (and elsewhere) it says "…in Montogmery form" but it's actually returning a `P::Uint`. Is that a copy&paste error or…?

Reply:
Remainder from before #163. Fixed.